nuitka/build/SconsInterface.py

```diff
@@ -287,6 +287,27 @@ def runScons(options, env_values, scons_filename):
         Tracing.flushStandardOutputs()
 
         with withEnvironmentVarsOverridden(env_values):
+            if source_dir:
+                # we wrote debug shell script only if build process called, not "--version" call.
+                scons_debug_script_name = "scons-debug.sh"
```

Comment:
If it's an else, make it an else, I dislike defaulting.

Reply:
> If it's an else, make it an else, I dislike defaulting.
OK, I'll keep that in mind for the future (I like defaulting  :( )